metron-platform/metron-solr/metron-solr-common/src/main/scripts/create_configset.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
This file needs to be added to `metron.spec`. RPM build fails with:

```
RPM build errors:
error: Installed (but unpackaged) file(s) found:
   /usr/metron/0.7.2/bin/create_configset.sh
    Macro %_prerelease has empty body
    Macro %_prerelease has empty body
    bogus date in %changelog: Tue Sep 25 2017 Apache Metron <dev@metron.apache.org> - 0.4.2
    bogus date in %changelog: Thu May 15 2017 Apache Metron <dev@metron.apache.org> - 0.4.0
    Explicit %attr() mode not applicaple to symlink: /root/BUILDROOT/metron-0.7.2-root/usr/metron/0.7.2/web/expressjs/node_modules/.bin/json2yaml
    Explicit %attr() mode not applicaple to symlink: /root/BUILDROOT/metron-0.7.2-root/usr/metron/0.7.2/web/expressjs/node_modules/.bin/mime
    Explicit %attr() mode not applicaple to symlink: /root/BUILDROOT/metron-0.7.2-root/usr/metron/0.7.2/web/expressjs/node_modules/.bin/yaml2json
    Installed (but unpackaged) file(s) found:
   /usr/metron/0.7.2/bin/create_configset.sh
RPM build errors encountered
```

Reply:
File added to metron.spec